### DISCUSSION TITLE
File picker doesn't show folders

### DISCUSSION THREAD

**Discussion by paulboudewijn** [directus/directus#8955] [Archive]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues) that already describes my problem.

### Describe the Bug

The "choose file from library" option in the image interface doesn't show the folder structure created through the file library. It only shows all uploaded images.

### To Reproduce

Create some folders  through the file library
Add some images to the folders
Create a collection with an image field
Add an image to the collection through the option "choose file from library"
See that the the images are present but folders are missing

### What version of Directus are you using?

v9.0.0-rc.98

### What version of Node.js are you using?

v16.10.0

### What database are you using?

MSSQL

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying Directus?

running locally

**Comment by azrikahar** [comment]
Moved this to discussion as it's not technically a bug 👍

> The "choose file from library" option in the image interface doesn't show the folder structure created through the file library. It only shows all uploaded images.

Currently there's no "folders" in Card/Table layouts, hence why it shows all images by default during picking, which I presume what you are seeing as well.

![chrome_xvHqDvq9de](https://user-images.githubusercontent.com/42867097/138024821-55a38581-832f-4bd0-b0f7-09e6428d5c26.png)

However, we can still do a filter (which is made a lot more easier with the recent Filter interface!) such as:

![chrome_7A8DYM7PfV](https://user-images.githubusercontent.com/42867097/138024739-ccacc850-3daf-49ab-bba4-2cc591678eb2.png)

_In the above example, the folder name was "Folder 1", hence the "contains 1" filter._

---

As for if you would like to have folders in these view, IIRC there were mockups for exactly this requirement on Discord last month, but it might not be a priority at the moment.

**Comment by rijkvanzanten** [comment]
Duplicate of https://github.com/directus/directus/discussions/4999
